src/components/wallet/Tokens.js

```diff
@@ -23,11 +23,11 @@ const StyledContainer = styled.div`
     }
 `;
 
-const Tokens = ({ tokens }) => {
+const Tokens = ({ tokens, onClick }) => {
     return (
         <StyledContainer>
             {tokens.map((token, i) => (
-                <TokenBox key={i} token={token}/>
+                <TokenBox key={i} token={token} onClick={onClick}/>
```

Comment:
I'm fixing this now, but just a reminder @Patrick1904 @marcinbodnar that using an index in an array for a `key` is an anti-pattern; if the order of the values in `tokens` changes, these child component's won't re-render because their keys haven't changed...

So imagine that next week we start showing tokens sorted by your balance from highest->lowest rather than a pure alphabetical sort; when a token's balance changes the list won't re-render to show the new order.

Or today, with the list sorted alphabetically, we run a background query to see if a user has new tokens and update the `tokens` store accordingly...you'll probably see the last token in the list rendered twice instead of seeing the new token inserted alphabetically in the right spot.

Keys should be unique to the item in the list -- in this case, I think `token.contractName` would be the right key since everything else could be a duplicated value :).

A good reference w/ examples of the pitfalls here: https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318